ocs_ci/ocs/resources/backingstore.py

```diff
@@ -39,7 +41,7 @@ def delete(self):
         # Todo: implement deletion assertion
 
 
-def backingstore_factory(request, cld_mgr, cloud_uls_factory):
+def backingstore_factory(request, cld_mgr, mcg_obj, cloud_uls_factory):
```

Comment:
this function is being used by tests.conftest.backingstore_factory fixture which passes `cld_mgr`